packages/vats/test/setup-vat-data.js

```diff
@@ -0,0 +1,25 @@
+/* global globalThis */
+import { makeFakeVirtualStuff } from '@agoric/swingset-liveslots/tools/fakeVirtualSupport.js';
+
+export const fakeVomKit = makeFakeVirtualStuff({
+  relaxDurabilityRules: false,
+});
+
+globalThis.WeakMap = fakeVomKit.vom.VirtualObjectAwareWeakMap;
+globalThis.WeakSet = fakeVomKit.vom.VirtualObjectAwareWeakSet;
+
+const { vom, wpm: watchedPromiseManager, cm: collectionManager } = fakeVomKit;
+globalThis.VatData = harden({
```

Comment:
I'm on my phone so it's hard to look up: what's the difference between this and the one in vat-data package?

This should have a comment on when it used to be used instead and why